From 65450a836ee9e0722a2d4c3327f52345eae293c6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Thu, 23 Dec 2021 17:18:16 +0100
Subject: [PATCH] build: Drop incorrect positional arg

Unlike other targets that take a name, i18n.merge_file() does not.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2078>
---
 data/meson.build                                      | 2 +-
 src/calendar-server/meson.build                       | 2 +-
 subprojects/extensions-app/data/meson.build           | 2 +-
 subprojects/extensions-app/data/metainfo/meson.build  | 2 +-
 subprojects/extensions-tool/src/templates/meson.build | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index f924fdf806..76ae45c93a 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -16,7 +16,7 @@ desktopconf.set('bindir', bindir)
 desktopconf.set('systemd_hidden', have_systemd ? 'true' : 'false')
 
 foreach desktop_file : desktop_files
-  i18n.merge_file('desktop',
+  i18n.merge_file(
     input: configure_file(
       input: desktop_file + '.in.in',
       output: desktop_file + '.in',
diff --git a/src/calendar-server/meson.build b/src/calendar-server/meson.build
index 7363282a59..8b4ef411c8 100644
--- a/src/calendar-server/meson.build
+++ b/src/calendar-server/meson.build
@@ -27,7 +27,7 @@ configure_file(
   install_dir: servicedir
 )
 
-i18n.merge_file('evolution-calendar.desktop',
+i18n.merge_file(
   input: 'evolution-calendar.desktop.in',
   output: 'evolution-calendar.desktop',
   po_dir: po_dir,
diff --git a/subprojects/extensions-app/data/meson.build b/subprojects/extensions-app/data/meson.build
index d7e7d4001c..4b601e8bd1 100644
--- a/subprojects/extensions-app/data/meson.build
+++ b/subprojects/extensions-app/data/meson.build
@@ -14,7 +14,7 @@ desktopconf.set('bindir', bindir)
 desktopconf.set('app_id', app_id)
 desktopconf.set('prgname', prgname)
 
-i18n.merge_file('desktop',
+i18n.merge_file(
   input: configure_file(
     input: base_id + '.desktop.in.in',
     output: desktop_file + '.in',
diff --git a/subprojects/extensions-app/data/metainfo/meson.build b/subprojects/extensions-app/data/metainfo/meson.build
index c4962c0576..a19bfa80a4 100644
--- a/subprojects/extensions-app/data/metainfo/meson.build
+++ b/subprojects/extensions-app/data/metainfo/meson.build
@@ -1,5 +1,5 @@
 metainfo = app_id + '.metainfo.xml'
-i18n.merge_file(metainfo,
+i18n.merge_file(
   input: base_id + '.metainfo.xml.in',
   output: metainfo,
   po_dir: po_dir,
diff --git a/subprojects/extensions-tool/src/templates/meson.build b/subprojects/extensions-tool/src/templates/meson.build
index 670e2bf448..d693bfaddb 100644
--- a/subprojects/extensions-tool/src/templates/meson.build
+++ b/subprojects/extensions-tool/src/templates/meson.build
@@ -4,7 +4,7 @@ template_metas = [
 ]
 template_deps = []
 foreach template : template_metas
-  template_deps += i18n.merge_file(template,
+  template_deps += i18n.merge_file(
     input: template + '.in',
     output: template,
     po_dir: po_dir,
-- 
GitLab

